addon/snapshot.js

```diff
@@ -95,26 +94,26 @@ export function percySnapshot(name, options) {
   let scope = options.scope;
 
   // Create a full-page DOM snapshot from the current testing page.
-  let domCopy = percyJQuery('html').clone();
-  let bodyCopy = domCopy.find('body');
-  let testingContainer = domCopy.find('#ember-testing');
+  let domCopy = document.documentElement.cloneNode(true);
```

Comment:
@rwwagner90 This is the big one that worries me, I don't have a good sense for what jQuery.clone is doing underneath and we need these things especially to be fully backwards compatible.

This apparently is what jQuery is doing underneath:
https://j11y.io/jquery/#v=1.11.2&fn=jQuery.clone

Reply:
@fotinakis according to http://youmightnotneedjquery.com/ `$(el).clone();` is the same as `el.cloneNode(true);`